pkg/clicmd/root.go

```diff
@@ -80,18 +81,25 @@ func appRoot() (string, error) {
 
 // parseCommand does an early parse of the command line and returns
 // what will ultimately be recognized as the command by cobra.
```

Comment:
Please update the comment to document the new return value.

Reply:
Updated comments on return value of parseCommand to include a boolean